puncover/renderers.py

```diff
@@ -25,6 +25,14 @@ def symbol_file(value):
     return value.get(collector.BASE_FILE, '__builtin')
 
 @jinja2.contextfilter
+def symbol_display_name_filter(context, value):
```

Comment:
why do we need any of the filter code? in `unmangle_cpp_names()` you make sure that every symbol has a `.display_name` already.